ibis/backends/base/__init__.py

```diff
@@ -40,6 +40,8 @@
     "impala": "hive",
     "pyspark": "spark",
     "polars": "postgres",
+    # TODO: not sure if this the closest match
```

Comment:
From the DB Developers I got the feedback, that `Oracle` should be the closest match. That said in case of doubt we generally tend to aim for compliance with the SQL standard. Still we also have some compatibility with other things like e.g. date formats of `MySql` ...

Reply:
@cpcloud

Reply:
Ok, let's leave it as oracle for now. Currently this is used in two major places:

1. printing generated sql
2. turning queries from other dialects into the backend dialect (exasol in this case)